README.md

```diff
@@ -8,39 +8,56 @@ will likely not be included in this repository, such as custom webhooks, etc.
 
 See also https://wikitech.wikimedia.org/wiki/Wikimedia_Cloud_Services_team/EnhancementProposals/Toolforge_Buildpack_Implementation
 
+## Setup dev environment
 
-### Setup dev environment
-#### Requirements
+### Requirements
 You will need to install:
 * [docker](https://www.docker.com/) -> as the container engine for everything.
 * [docker-compose](https://docs.docker.com/compose/) -> for harbor (docker registry).
 * [minikube](https://minikube.sigs.k8s.io/docs/) -> as dev kubernetes installation.
 
-#### Run harbor
-You can install it with the helper script:
-* `utils/get_harbor.sh`
-After that you can use docker-compose to run the whole harbor system (see the output of the script for the exact command).
-
-Once harbor is running, you will need to make sure that it's setup (user project created and such) so run:
-- `utils/setup_harbor.py`
-
 ### Setup minikube
 We will need to get a specific k8s version (1.21.8 is the current toolforge version when writing this, you might want to double check):
  - `minikube start --kubernetes-version=v1.21.8`
 
+### Find your Docker IP
+You will need to reach the Docker daemon on your host from inside Minikube using
+an IP (this is a limitation of Tekton that doesn't support hostnames unless you
+have a valid SSL certificate).
+
+The IP to use will vary depending on your machine, but you can usually find it
+by running:
+- `minikube ssh`
+- `grep host.minikube.internal /etc/hosts`
+
+### Setup Harbor
+You can install Harbor with the helper script:
+- `utils/get_harbor.sh DOCKER_IP` (use the Docker IP from the previous step)
```

Comment:
Hmm... why not just `export HARBOR_IP=...` and let all the scripts use the env var?

Reply:
(I understood that was your intention no?)

Reply:
I'm sure I had a reason in mind when I wrote this, but I can't remember it anymore! 😆 I'll change this to use an env var!